src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -114,8 +113,8 @@ public ResponseEntity<Object> updateSubmission(
             //      sophisticated - this is covered by ROE-1415. For now though, it's only necessary to check if
             //      all validation checks should be run, i.e. if the user is calling the PUT end-point from the
             //      'check your answers' screen.
-            if (isValidationEnabled && isValidationRequired(overseasEntitySubmissionDto)) {
-                var validationErrors = overseasEntitySubmissionDtoValidator.validate(
+            if (isValidationEnabled) {
```

Comment:
the comment on lines 112 to 115 can be removed now, as it was explaining the validation process before we implemented this new partial process.

Reply:
Removed